statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1322,67 +1738,78 @@ def cov_params(self):
         Notes
         -----
         Covariance of vec(B), where B is the matrix
-        [params_for_deterministic_terms, A_1, ..., A_p] with the shape
-        (K x (Kp + number_of_deterministic_terms))
+
+        [params_for_deterministic_terms, A_1, ..., A_p] with the shape (q, q)
+        where q = neqs x (neqs x k_ar + number_of_deterministic_terms)
+
         Adjusted to be an unbiased estimator
+
         Ref: Lütkepohl p.74-75
         """
-        z = self.ys_lagged
-        return np.kron(scipy.linalg.inv(np.dot(z.T, z)), self.sigma_u)
+        z = self.endog_lagged
+        return np.kron(scipy.linalg.inv(np.dot(z.T, z)), self.cov_resid)
 
     def cov_ybar(self):
+        """
+        Deprecated.  Use cov_sample_mean.
+        """
+        return self.cov_sample_mean()
+
+    def cov_sample_mean(self):
         r"""Asymptotically consistent estimate of covariance of the sample mean
```

Comment:
"covariance of the sample mean" seems misleading.  The sample mean is known.  Maybe the covariance of the estimated process mean?

Reply:
It is the covariance of the sample mean estimator - cov_sample_mean_estimator seems excessive,